templates/base.html

```diff
@@ -87,29 +87,43 @@
 
     <a href="{{ url('home') }}" class="logo">{{ _('Mozilla Developer Network') }}</a>
 
-    <div class="header-login">
+    <div id="nav-sec">
         {% if not is_sphinx %}
-            {% include "includes/login.html" %}
+
+            {% if waffle.flag('header_tools') %}
+                  <ul>
+                  {% if user.is_authenticated() %}
+                      <li class="nav-tools"><a href="{{ wiki_url('Mozilla/Connect') }}">{{ _('Tools') }}<i aria-hidden="true" class="icon-caret-down"></i></a>
```

Comment:
None of these `wiki_url` arguments seem to match with their link labels? Mis-commit?